.github/workflows/test.yaml

```diff
@@ -103,9 +103,9 @@ jobs:
       - name: Run Isolated Tests
         run: pnpm test:e2e
         env:
-          TESTNET_WALLET_PVT_KEY: ${{ secrets.TESTNET_WALLET_PVT_KEY }}
+          # TESTNET_WALLET_PVT_KEY: ${{ secrets.TESTNET_WALLET_PVT_KEY }}
```

Comment:
Note to restore testnet and disable devnet once it is supported in both environments.

Reply:
devnet e2e passing so enabling testnet